tasks/quality_gates.py

```diff
@@ -260,3 +286,58 @@ def manual_threshold_update(self, filename="static_gate_report.json"):
     github = GithubAPI()
     pr_url = update_quality_gates_threshold(self, metric_handler, github)
     notify_threshold_update(pr_url)
+
+
+@task()
+def exception_threshold_bump(ctx):
+    """
+    When a PR is exempt of static quality gates, they have to use this invoke task to adjust the quality gates thresholds accordingly to the exempted added size.
+
+    Note: This invoke task must be run on a pipeline that has finished running static quality gates
+    :param ctx:
+    :return:
+    """
+    current_branch_name = ctx.run("git rev-parse --abbrev-ref HEAD", hide=True).stdout.strip()
+    ancestor_commit = get_common_ancestor(ctx, "HEAD")
+    repo = get_gitlab_repo()
+    with tempfile.TemporaryDirectory() as extract_dir, ctx.cd(extract_dir):
+        with open(f"{extract_dir}/gate_archive.zip", "wb") as f:
+            try:
+                f.write(repo.artifacts.download(ref_name=current_branch_name, job="static_quality_gates"))
+            except gitlab.exceptions.GitlabGetError as e:
+                print(
+                    color_message(
+                        "[ERROR] Unable to fetch the last artifact of the static_quality_gates job. Details :", "red"
+                    )
+                )
+                print(repr(e))
+                return
```

Comment:
```suggestion
                raise Exit(code=1)
```
If we want to ignore the error I suppose we should say it, we should not silent the error otherwise.